app/src/main/java/edu/uc/hickmadc/customnotifications/ui/main/dialogfragment.kt

```diff
@@ -5,30 +5,58 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.Button
+import android.widget.ImageButton
+import android.widget.TextView
 import androidx.fragment.app.DialogFragment
 import androidx.fragment.app.FragmentContainer
 import edu.uc.hickmadc.customnotifications.R
+import edu.uc.hickmadc.customnotifications.dto.Notification
 import java.io.FileDescriptor
 import java.io.PrintWriter
 import java.util.zip.Inflater
 
 
+class dialogfragment : DialogFragment() {
 
-class dialogfragment: DialogFragment() {
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
-    ): View?{
+    private lateinit var rootView: View
+    private lateinit var notificationTitleText: TextView
+    private lateinit var notificationSubtext: TextView
+    private lateinit var notificationText: TextView
```

Comment:
These are variables I created to hold the ui components of your pop up dialog box which allows you to create a new custom notification. The rootView variable already was created, but I extracted it out of a method to make it a class variable

Reply:
Are these used anywhere yet?  If not, I urge caution on implementing partially complete code.  It can give the wrong indication that these fields actually are used.

Only complete features, which meet the team's Definition of Done, should be merged to Master.